### PR TITLE
chore(env): different fetch interval for messages in suite and mobile

### LINKED ISSUE #10276: Inspect: Fetching of remote JWS config failed: TypeError: Network request failed
Inspect and resolve what causes
`Fetching of remote JWS config failed: TypeError: Network request failed`

while fetching messages config

make the possible error appear less by making calls less often.

### PR DESCRIPTION
**Pull request by vytick**
Add possibility to have different messages check and fetch intervals on native and suite.

Resolve #10276

### CODE REVIEW

File: packages/env-utils/src/envUtils.native.ts [hunk 1]
```diff
@@ -73,6 +73,15 @@ const getOsNameWeb = () => '';
 
 const getOsFamily = (): 'Linux' => 'Linux';
 
+/*
+ * On the app launch and then about once a minute (MESSAGES_FETCH_INTERVAL), a new config tries to fetch.
+ * MESSAGES_CHECK_INTERVAL is interval for checking if MESSAGES_FETCH_INTERVAL already expired.
+ * in miliseconds
+ */
+
+const MESSAGES_FETCH_INTERVAL = 600_000;
```
Comment (Nodonisko) [inline packages/env-utils/src/envUtils.native.ts]: Please always suffix name with units like `IN_MS` `IN_S` etc.

Comment (Nodonisko) [inline packages/env-utils/src/envUtils.native.ts]: This doesn't belong to env-utils, this should be in message package. You can do `MESSAGES_CHECK_INTERVAL = isMobile() ? 100000 : 50000` etc

Comment (vytick) [reply packages/env-utils/src/envUtils.native.ts]: I have followed previous naming convention. I'll update it

Comment (vytick) [reply packages/env-utils/src/envUtils.native.ts]: 3e7398770

Comment (vytick) [reply packages/env-utils/src/envUtils.native.ts]: 3e7398770

**Review by Nodonisko — APPROVED** [review]

**Review by matejkriz — APPROVED** [review]
10 minutes seems long to me because it means that if there's a major problem, like an attack in progress, the user could use the app for 10 minutes before finding out about it, and in that time they could send something. But for now it may be OK and in time we may consider shortening that interval again.

But I believe that the cost of subsequent request is really small, the response is cashed and client is not fetching the whole file, just checking for changes...

**Comment by Nodonisko** [comment]
@matejkriz It's not true that cost is small, on mobile where do don't have enough of raw power, validating of message can took 1 - 2s on slower androids and during this time app is basically unresponsive. There are ways how to improve it like react-native-quick-crypto but I don't have time to look into this right now.

**Comment by matejkriz** [comment]
> @matejkriz It's not true that cost is small, on mobile where do don't have enough of raw power, validating of message can took 1 - 2s on slower androids and during this time app is basically unresponsive. There are ways how to improve it like react-native-quick-crypto but I don't have time to look into this right now.

I see, I meant the cost of the network request, it didn't occur to me that the validation could be so slow on a mobile. 

If the validation is slow but decoding of JWS is ok, there can be an low hanging improvement to completely skip processing of the config (including verification) if the config.sequence hasn't increased.

**Comment by vytick** [comment]
That sounds like a good optimisation. Maybe we can make an issue for this for after release.

**Pull request merged**